src/js/components/SetUpAccount/AddContactsFromGoogleButton.jsx

```diff
@@ -235,6 +243,25 @@ class AddContactsFromGoogleButton extends Component {
     });
   }
 
+  cordovaSignOn () {
```

Comment:
My hope is to use an entirely different component when on a device, in most cases, but there might be some scenarios in the future where we also want to support importing from Gmail. But I consider this a low priority.

Reply:
Those changes were to support Sign In With Google in Cordova, which would take weeks more to finish.  As is, they don't hurt anything, and took a day+ to figure out.  I would only copy a class to make a Cordova version as a last resort.